docs/rules/html-closing-bracket-spacing.md

```diff
@@ -0,0 +1,83 @@
+# require or disallow a space before tag's closing brackets (html-closing-bracket-spacing)
+
+- :wrench: The `--fix` option on the [command line](http://eslint.org/docs/user-guide/command-line-interface#fix) can automatically fix some of the problems reported by this rule.
+
+This rule enforces space style before `>` of tags.
```

Comment:
`This rule enforces consistent spacing style before closing brackets `>` of elements.`